DigitalLearningSolutions.Web/Scripts/trackingSystem/emailDelegates.ts

```diff
@@ -27,5 +33,11 @@ const selectAllButton = document.getElementById('select-all-button') as HTMLButt
 const deselectAllButton = document.getElementById('deselect-all-button') as HTMLButtonElement;
 
 selectAllForm.addEventListener('submit', (e) => e.preventDefault());
-selectAllButton.addEventListener('click', selectAll);
-deselectAllButton.addEventListener('click', deselectAll);
+selectAllButton.addEventListener('click', () => {
+  selectAll();
+  alertResultCount();
+});
+deselectAllButton.addEventListener('click', () => {

```

Comment:
Minor, but missing an empty line above.

Also, can we wrap all this with setUpSelectAndDeselectButtons() function and call that at the top of the page (so devs don't have to scroll through the whole file to see what is happening on page load).